matlabCode/createBIDS_balloonanalog_events_tsv.m

```diff
@@ -0,0 +1,40 @@
+%% Template Matlab script to create an BIDS compatible sub-01_ses-01_task-balloonanalogrisktask_run-01_events.tsv file
+% This example lists all required and optional fields.
+% When adding additional metadata please use camelcase
+% 
+%
+% anushkab, 2018
+%%
+clear all
+root_dir = '../';
+project_label = 'templates';
+sub_id = '01';
+ses_id = '01';
+task_id = 'balloonanalogrisktask';
+
+acquisition = 'func';
+run_id='01';
+
+
+events_tsv_name = fullfile(root_dir,project_label,...
+  ['sub-' sub_id],...
+              ['ses-' ses_id],acquisition,...
+              ['sub-' sub_id ...
+              '_ses-' ses_id ...
+              '_task-' task_id ...
+              '_run-' run_id '_events.tsv']);
+%% make an _events table and save 
+onset = [0.061]'; % onsets in seconds
+duration = [0.772]'; % duration in seconds (measured from onset)
+trial_type={'pumps_demean'};  %Primary categorisation of each trial to identify them as instances of the experimental conditions
+cash_demean={'n/a'}; %
```

Comment:
Templates are supposed to align with the specifications, rather than with a specific example. Therefore, should have the following fields, as in [BIDS spec](https://docs.google.com/document/d/1HFUkAEE-pB-angVcYe6pf_-fVf4sCpOHKesUvfb8Grc/edit#)

onset | REQUIRED.  Onset (in seconds) of the event  measured from the beginning of the acquisition of the first volume in the corresponding task imaging data file.  If any acquired scans have been discarded before forming the imaging data file, ensure that a time of 0 corresponds to the first image stored. In other words negative numbers in “onset” are allowed.
-- | --
duration | REQUIRED. Duration of the event (measured  from onset) in seconds.  Must always be either zero or positive. A "duration" value of zero implies that the delta function or event is so short as to be effectively modeled as an impulse.
trial_type | OPTIONAL. Primary categorisation of each trial to identify them as instances of the experimental conditions. For example: for a response inhibition task, it could take on values "go" and "no-go" to refer to response initiation and response inhibition experimental conditions.
response_time | OPTIONAL. Response time measured in seconds. A negative response time can be used to represent preemptive responses and “n/a” denotes a missed response.
stim_file | OPTIONAL. Represents the location of the stimulus file (image, video, sound etc.) presented at the given onset time. There are no restrictions on the file formats of the stimuli files, but they should be stored in the /stimuli folder (under the root folder of the dataset; with optional subfolders). The values under the stim_file column correspond to a path relative to “/stimuli”. For example “images/cat03.jpg” will be translated to “/stimuli/images/cat03.jpg”.
HED | OPTIONAL. Hierarchical Event Descriptor (HED) Tag. See Appendix III for details.